Calculator.py

```diff
@@ -0,0 +1,134 @@
+import re
+import math
+from statistics import mean
+
+from pip._vendor.msgpack.fallback import xrange
+
+
+class Calculator:
+    def __init__(self, exercise):
+        self.__operations_regex_dict = {1: r'\(\d+[!%^+/*\-~$&@]\d+\)+',
+                                        2: r'\!',
+                                        3: r'[$&@]',
+                                        4: r'%',
+                                        5: r'\^',
+                                        6: r'[/*]',
+                                        7: r'[-+]'}
+        self.exercise = exercise.strip()
+        self.solved = ''
+        print(self.arrange)
+
+    @property
+    def arrange(self):
+        self.solved = re.search(r'(!\d+)|(([~!]\d+)?\d+[-()%^/*+$&@]([~!]\d+)?\d+)', self.exercise)
+        if self.solved is None:
+            return self.exercise
+        exercise = self.exercise
+        for operation in self.__operations_regex_dict.values():
+            if self.__operations_regex_dict[1] == operation:
+                reg = str(operation)
+                exp = re.search(reg, exercise)
+            elif self.__operations_regex_dict[2] == operation:

```

Comment:
blank line between each logic block (at the end of every if, elif, else, for and while)